lib/modules/platform/github/massage-markdown-links.spec.ts

```diff
@@ -87,4 +88,20 @@ describe('modules/platform/github/massage-markdown-links', () => {
       expect(massageMarkdownLinks(input)).toEqual(output);
     }
   );
+
+  it('replace url contains disappear text inline', () => {
```

Comment:
This description does not match the test code. Would something like this work?

```suggestion
  it('replace @user with togithub.com link to user profile and replace foo/bar#1 with togithub.com link, ', () => {
```

Reply:
sorry i changed the code there but not the name, thx !